app/services/claims/injection_channel.rb

```diff
@@ -2,7 +2,6 @@ module Claims
   class InjectionChannel
     def self.for(claim)
       return 'cccd_development' if claim.nil?
-      return 'cccd-k8s-injection' if Settings.aws&.response_queue&.match?('laa-get-paid')
       claim.agfs? ? 'cccd_ccr_injection' : 'cccd_cclf_injection'
```

Comment:
Is it worth combining these into a single, `cccd-injection` channel now?  We are only getting 1 or 2 per in each channel.

Reply:
I rather like the separate channels as it makes searching for specific errors related to ccr or cclf clearer/easier. But could swap this out for single channel in the near future.